playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -46,12 +46,16 @@
 
   # Replace dc/docker-registry certificate secret contents if set.
   - block:
+    - name: Load lib_openshift modules
+      include_role:
+      name: lib_openshift
+
     - name: Retrieve registry service IP
-      command: >
-        {{ openshift.common.client_binary }} get service docker-registry
-        -o jsonpath='{.spec.clusterIP}'
-        --config={{ mktemp.stdout }}/admin.kubeconfig
-        -n default
+      oc_service:
+        namespace: default
+        name: docker-registry
+        state: list
+        kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
```

Comment:
It's documented in the module docs. A recent update to the modules added the feature to copy them by default.